NOTICE

```diff
@@ -163,6 +164,7 @@ This project includes:
   Jetty :: XML utilities under Apache Software License - Version 2.0 or Eclipse Public License - Version 1.0
   Jetty Server under Apache Software License - Version 2.0 or Eclipse Public License - Version 1.0
   Jetty Utilities under Apache Software License - Version 2.0 or Eclipse Public License - Version 1.0
+  JLine under BSD
   JSch under BSD
   json4s-ast under ASL
   json4s-core under ASL
```

Comment:
This is strange, I wonder why the test passed on Travis. Did you see the problem on your local machine?